### PR TITLE
KafkaMonitor logger additivity in 0.8.2.2

### PR DESCRIPTION
**Pull request by hackerwin7**
Correct the log4j additivity for com.linkedin.kmf.core.KafkaMonitor.

### CODE REVIEW

File: config/log4j.properties [hunk 1]
```diff
@@ -19,7 +19,7 @@ log4j.appender.kafkaClientAppender.layout=org.apache.log4j.PatternLayout
 log4j.appender.kafkaClientAppender.layout.ConversionPattern=[%d] %p %m (%c)%n
 
 log4j.logger.com.linkedin.kmf.core.KafkaMonitor=INFO, stdout
-log4j.additivity.kmf.tools.KafkaMonitor=false
+log4j.additivity.com.linkedin.kmf.core.KafkaMonitor=false
```
Comment (lindong28) [inline config/log4j.properties]: Thanks for catching this. Can you change it to `com.linkedin.kmf.KafkaMonitor`?

Comment (hackerwin7) [reply config/log4j.properties]: Thanks your comment! the KafkaMonitor in branch 0.8.2.2 is in original package `com.linkedin.kmf.core`, It seems that I should modify the package from `com.linkedin.kmf.core` to `com.linkedin.kmf`. Is it sounds good for this?

Comment (lindong28) [reply config/log4j.properties]: Ah I see. This change is for 0.8.2.2. Sounds good.

Comment (lindong28) [reply config/log4j.properties]: And thanks for the patch!

**Pull request merged**